### ISSUE TITLE
train accuracy

### ISSUE THREAD

**Issue by abhishekvarma23** [qqwweee/keras-yolo3#371]
How to calculate training accuracy at every epoch?

**Comment by shanytc** [comment]
pass the accuracy metric in the model.compile() function.
Or use BlastML library, it includes it already. (pip install BlastML) or go to github.com/shanytc/BlastML/

**Comment by abhishekvarma23** [comment]
> github.com/shanytc/BlastML/
how to use model.compile for custom loss. in our case yolo_loss

**Comment by ashish-roopan** [comment]
Adding accuracy metric in model.complie() doesnt calculate accuracy .It always prints 0.000 as accuracy.

**Comment by wuxiangchao** [comment]
model.complie(...,...,metrics =["acc"])

**Comment by zhangyufei1995** [comment]
have you solve it?

**Comment by T1mu** [comment]
> Adding accuracy metric in model.complie() doesnt calculate accuracy .It always prints 0.000 as accuracy.

As with the problems I have encountered, have you solved it?

**Comment by ashish-roopan** [comment]
https://github.com/ashish-roopan/mAP.

You need to calculate mAP after you have tested your model with test set .

Use this repo. You only have to give the ground truth and predicted truth in the required format.
